core/src/test/java/hudson/UtilTest.java

```diff
@@ -352,6 +355,7 @@ public void testDeleteContentsRecursive() throws Exception {
     }
 
     @Test
+    @Retry
```

Comment:
This just duplicates the Surefire feature which is already enabled in the repo.

If the test is flaky enough that it is failing for you five times in a row, just use

```java
@Ignore("TODO fails frequently in CI")
```

and leave it for a rainy day for someone to figure out why it was failing and how to fix it, or to decide it is worthless and just delete it.

Reply:
The surefire feature is a global setting, while this one is localized.